### PR TITLE
Fix typo in example docs

### PR DESCRIPTION
**Pull request by benxgao**

### CODE REVIEW

File: examples/bootstrap/css-gridish/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Carbon Design Grid
+# Bootstrap Design Grid
 
 An example of [CSS Gridish](../../../README.md) generating CSS Grid code, fallback flexbox code, dev documentation, and Sketch files based on the [Carbon Design System grid.](https://github.com/carbon-design-system/carbon-components/blob/master/src/globals/grid/_grid.scss#L29-L31)
```
Comment (seejamescode) [inline examples/bootstrap/css-gridish/README.md]: This line needs to be updated also

File: examples/material/css-gridish/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Carbon Design Grid
+# Material Design Grid
 
 An example of [CSS Gridish](../../../README.md) generating CSS Grid code, fallback flexbox code, dev documentation, and Sketch files based on the [Carbon Design System grid.](https://github.com/carbon-design-system/carbon-components/blob/master/src/globals/grid/_grid.scss#L29-L31)
```
Comment (seejamescode) [inline examples/material/css-gridish/README.md]: This line needs to be updated also

File: examples/bootstrap/css-gridish/README.md [hunk 3]
```diff
@@ -144,4 +144,4 @@ Thanks to flexbox’s wrapping functionality, nodes that specify rows are not ne
 
 ### What happens in the legacy implementation if I specify the column width for one breakpoint, but not the next larger breakpoint?
 
-To maintain a mobile-first opinion, column widths will scale to the next breakpoint if not specified. This means that a `.bootstrap-grid__col--sm--1` be the size of `.bootstrap-grid__col----2` if no `` class was declared.
```
Comment (seejamescode) [inline examples/bootstrap/css-gridish/README.md]: Hey @benxgao, these lines are automatically generated. Update the prefix in `css-gridish.json` of this directory. Then run `npm run build:examples` in the root directory of the project.

File: examples/material/css-gridish/README.md [hunk 4]
```diff
@@ -150,4 +150,4 @@ Thanks to flexbox’s wrapping functionality, nodes that specify rows are not ne
 
 ### What happens in the legacy implementation if I specify the column width for one breakpoint, but not the next larger breakpoint?
 
-To maintain a mobile-first opinion, column widths will scale to the next breakpoint if not specified. This means that a `.material-grid__col--xsmall--1` be the size of `.material-grid__col--small--2` if no `small` class was declared.
```
Comment (seejamescode) [inline examples/material/css-gridish/README.md]: Hey @benxgao, these lines are automatically generated. Update the prefix in `css-gridish.json` of this directory. Then run `npm run build:examples` in the root directory of the project.

**Comment by benxgao** [comment]
Thanks @seejamescode , just made those changes.

**Pull request merged**